ui/admin/tests/acceptance/credential-store/credentials/delete-test.js

```diff
@@ -71,6 +79,10 @@ module(
         return this.server.schema.credentials.where({ type: 'ssh_private_key' })
           .length;
       };
+      getJSONCredentialCount = () => {
```

Comment:
Should this also be lowercase?

Reply:
I didn't lowercase it because JSON is an acronym and it isn't the beginning of the variable name. Happy to change it to `getJsonCredentialCount` if that is better.